queries/nhs-choices/monitoring.json

```diff
@@ -0,0 +1,12 @@
+{
+  "data-set": {
+    "data-group": "nhs-choices", 
+    "data-type": "monitoring"
+  }, 
+  "entrypoint": "performanceplatform.collector.pingdom", 
+  "options": {}, 
+  "query": {
+    "name": "xxxxxxxx"
```

Comment:
I know this is not related to this commit, but why is this `xxxxxxxx`. That sounds broken, maybe @zilnhoj will know, if this isn't meant to be working then it shouldn't be committed.